modules/installation-creating-aws-worker-machineset.adoc

```diff
@@ -0,0 +1,150 @@
+// Module included in the following assemblies:
```

Comment:
We're no longer in aws land so how about a filename of installation-creating-ocp-worker-machineset.adoc

Reply:
Depending on what Trevor decides, I might be pulling this file from the PR. For now, it's a UPI-only task, and I'll update the filename.